src/utils/useCollection/index.ts

```diff
@@ -1,52 +1,48 @@
 import { useState, useEffect } from 'react'
-import firestore from '#utils/useFirestore'
+import firestore from '#utils/firestore'
```

Comment:
Why was the naming convention changed?

Reply:
it's not a hook, on the convention did not change, it was fixed

Reply:
Lets make it one then 
https://usehooks.com/useFirestoreQuery/

Reply:
This wont block anything from being merged, but can be a TODO for sure

Reply:
this hook is essentially what is implemented in `useCollection` logic-wise, I found that there is no use for a generic access to firestore............ _for now_

Reply:
useCollection is specific, I would rather there is a `useFirebase` and is **composed** in more specific Custom Hooks like `useCollection`, `useCharacters`, `useWhateverFirebaseData`, or whatever. Likely the general Firebase logic was put into `useCollection` due to when it was written it being the only Firebase Query

Reply:
There are two query style hooks so far - `useDocument` and `useCollection` - useCollection can probably be turned into a `useFirestoreQuery` hook keeping all the same logic.

`useDocument` could then be re-written to utilize `useFirestoreQuery` under-the-hood, but I think it will still be a useful abstraction for just grabbing one document by it's id - though I can understand not wanting it at all :stuck_out_tongue: